src/server/index.ts

```diff
@@ -1,38 +1,53 @@
 /* eslint-disable react-hooks/rules-of-hooks */
-const {
+/**
+ * Polyfill for dataloader.
+ * TODO: Remove it once this is fixed: https://github.com/graphql/dataloader/issues/249
```

Comment:
The issue above is already closed, it seems to be fixed here: https://github.com/graphql/dataloader/pull/293

Reply:
nice, we can check it on a later pr